.snippets/code/tutorials/eth-api/hardhat-start-to-end/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+// prettierrc.cjs or .prettierrc.js
```

Comment:
this can be deleted. there is already `.editorconfig` file in the root docs directory